mr/src/main/java/org/apache/iceberg/mr/Catalogs.java

```diff
@@ -56,8 +60,10 @@
 public final class Catalogs {
   private static final Logger LOG = LoggerFactory.getLogger(Catalogs.class);
 
-  private static final String HADOOP = "hadoop";
-  private static final String HIVE = "hive";
+  public static final String DEFAULT_CATALOG = "default";
```

Comment:
In Spark, the default catalog is named `default_iceberg`. Could we use that here as well?